mmselfsup/models/backbones/mae_cls_vit.py

```diff
@@ -0,0 +1,110 @@
+import torch
```

Comment:
Can `mae_cls_vit.py` and `mae_pretrain_vit.py` be merged into one file? Also, can `MAEClsViT` be written as a more general form so that downstream linear probing and fine-tuning can be equally applicable to other methods with the same backbone.

Reply:
Thank you for your valuable comments.  It's not convenient to merge `mae_cls_vit.py` and `mae_pretrain_vit.py` into one file, since they share different `forward` func. For `MAEClsViT`, it has specific initialization config, so it's not appropriate to rewrite it in a more general form.